plugin.xml

```diff
@@ -33,6 +33,13 @@
         <source-file src="src/ios/RCPurchaserInfo+CDVPurchasesPlugin.m" />
         <header-file src="src/ios/SKProduct+CDVPurchasesPlugin.h" />
         <source-file src="src/ios/SKProduct+CDVPurchasesPlugin.m" />
-        <framework src="src/ios/Purchases.framework" embed="true" custom="true" />
+        <podspec>
```

Comment:
I believe the problem is <podspec> is only supported by cordova-ios 5.0.0 Phonegap build latest supported is 4.5.5 Can you update it to the old way? Or I can fork this and do it myself.

<framework src="Purchases" spec="~> 2.3.0" />